test_conformance/half/Test_vStoreHalf.cpp

```diff
@@ -783,18 +817,15 @@ int Test_vStoreHalf_private( cl_device_id device, f2h referenceFunc, d2h doubleR
                 fchk.aspace = addressSpaceNames[addressSpace];
                 dchk.aspace = addressSpaceNames[addressSpace];
 
-                cl_uint pattern = 0xdeaddead;
-                memset_pattern4( gOut_half, &pattern, BUFFER_SIZE/2);
-
-                error = clEnqueueWriteBuffer(gQueue, gOutBuffer_half, CL_FALSE,
-                                             0, count * sizeof(cl_half),
-                                             gOut_half, 0, NULL, NULL);
+                error = RunKernel(device, resetKernel, gInBuffer_single, gOutBuffer_half,
+                                  count, 0);
```

Comment:
Do you see the same benefits using [clEnqueueFillBuffer](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#clEnqueueFillBuffer) on your device or is a kernel required?

Reply:
I have not tested with clEnqueueFillBuffer. I can explore using this as opposed to a kernel.